lib/manager/dockerfile/extract.ts

```diff
@@ -34,16 +34,23 @@ export function splitImageParts(currentFrom: string): PackageDependency {
 
 export function getDep(
   currentFrom: string,
-  specifyReplaceString = true
+  specifyReplaceString = true,
+  config: ExtractConfig
```

Comment:
You can't have required parameter after optional parameter. This won't work. Also this function is called by a lot other managers, song they will fail too.

Reply:
Can I get the config without passing it from all callers of `getDep`?

Reply:
nope, not yet. 🤷‍♂️ better only pass alias object.

Reply:
I changed `config` to `aliases`, moved it in front of `specifyReplaceString` and changed all calls of `getDep` to pass aliases.